lib/Dialect/ESI/cosim/MtiPliStub/DummySvDpi.cpp

```diff
@@ -0,0 +1,187 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+// Based on code from Verilator:
+//
+// Copyright 2009-2020 by Wilson Snyder. This program is free software; you can
+// redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
```

Comment:
I'm sorry, but importing this code is a non-starter.  We need to maintain a consistent license here.  Is wilson willing to contribute the code under the llvm license?

Reply:
Yeah, I was concerned about that. I'll contact Wilson.

Reply:
thx!